crates/sui-transactional-test-runner/src/simulator_persisted_store.rs

```diff
@@ -705,6 +705,26 @@ impl ReadStore for PersistedStoreInnerReadOnlyWrapper {
     }
 }
 
+impl RestStateReader for PersistedStoreInnerReadOnlyWrapper {
+    fn get_transaction_checkpoint(
+        &self,
+        _digest: &TransactionDigest,
+    ) -> sui_types::storage::error::Result<Option<CheckpointSequenceNumber>> {
+        todo!()
```

Comment:
A reminder here, in case it can be done now.

Reply:
This is in test-only code.